sonic-slave-bullseye/Dockerfile.j2

```diff
@@ -62,6 +62,8 @@ RUN apt-get update && apt-get install -y \
         dh-exec \
         kmod \
         libtinyxml2-dev \
+	python \
```

Comment:
This is for SDK compilation correct?

Reply:
Correct. Looks like I need to fix an indention here.